src/server/modes/charts/plugins/datalens/preparers/pie/__tests__/pie.test.ts

```diff
@@ -36,8 +36,8 @@ describe('preparePie', () => {
         }));
 
         expect(items).toEqual([
-            {color: '#4DA2F1', colorValue: '2'},
-            {color: '#FF3D64', colorValue: '1'},
+            {color: '#4DA2F1', colorValue: '1'},
```

Comment:
Why do you need to edit tests? They should have kept the old behavior, shouldn't they?